libs/designer/src/lib/ui/panel/panelRoot.tsx

```diff
@@ -61,6 +61,13 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
     [currentPanelMode]
   );
 
+  const customPanelLocations: CustomPanelLocation[] = [];
+  for (const customPanelLocation of customPanelLocations) {
```

Comment:
you're initializing customPanelLocations to an empty array above so this will never get ran?

Reply:
I will be passing this in as a Designer prop, this was only temporary, will update in next commit